README.md

```diff
@@ -216,6 +213,7 @@ osmt.security.role.curator=ROLE_Osmt_Curator
 osmt.security.role.view=ROLE_Osmt_View
 osmt.security.scope.read=SCOPE_osmt.read
 ```
+* NOTE: if app.enableRoles=false, all endpoints will be exposed!!
```

Comment:
This feels very dramatic. Can you say what you mean to say here in a more measured and objective way?

Reply:
Perhaps this line should read:
`if app.enableRoles=false, all endpoints will be accessible by any authenticated user.`

Is that what was meant?

Reply:
Done deal.